js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+        return this.parseTicker (data, symbol);
+    }
+
+    parseTicker (data, market = undefined) {
+        let timestamp = data['at'];
+        const ticker = data['ticker'];
+        if (typeof timestamp === 'string') {
+            if (timestamp.length > 0) {
+                timestamp = this.parse8601 (timestamp);
+            }
+        }
+        return {
+            'symbol': market,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeFloat (ticker, 'high'),
+            'low': this.safeFloat (ticker, 'low'),
+            'bid': this.safeFloat (ticker, 'buy'),
+            'bidVolume': undefined,
+            'ask': this.safeFloat (ticker, 'sell'),
+            'vwap': undefined,
+            'open': undefined,
+            'close': this.safeFloat (ticker, 'last'),
+            'last': this.safeFloat (ticker, 'last'),
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeFloat (ticker, 'volume'),
+            'quoteVolume': undefined,
+            'info': ticker,
+        };
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        let response = await this.publicGetOrderBook (this.extend ({
+            'market': symbol,
+            'limit': limit,
+        }, params));
+        const orderbook = this.safeValue (response, 'data');
+        return this.parseOrderBook (orderbook, orderbook['timestamp']);
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        let response = await this.publicGetTrades (this.extend ({
+            'market': symbol,
+            'limit': limit,
+        }, params));
+        let data = this.safeValue (response, 'data');
+        return this.parseTrades (data, symbol, since, limit);
+    }
+
+    parseTrade (trade, market = undefined) {
+        let side = undefined;
+        if (trade['side'] === 'bid') {
+            side = 'buy';
+        } else if (trade['side'] === 'ask') {
+            side = 'sell';
+        }
+        return {
+            'info': trade,
+            'timestamp': trade['created_on'],
+            'datetime': trade['created_at'],
+            'symbol': trade['market'],
+            'id': trade['id'],
+            'order': undefined,
+            'type': 'limit',
+            'takerOrMaker': undefined,
+            'side': side,
+            'price': this.safeFloat (trade, 'price'),
+            'amount': this.safeFloat (trade, 'volume'),
+            'cost': undefined,
+            'fee': undefined,
+        };
+    }
+
+    async fetchTime (params = {}) {
+        let response = await this.publicGetTimestamp ();
+        return response['data'];
+    }
+
+    async fetchTradingFees (params = {}) {
+        let response = await this.publicGetFeesTrading ();
+        return response['data'];
```

Comment:
We will need to parse these trading fees to a unified format, see an example here: https://github.com/ccxt/ccxt/blob/master/js/bitfinex.js#L409